src/test/java/com/example/universe/simulator/entityservice/unit/controllers/PlanetControllerTest.java

```diff
@@ -0,0 +1,311 @@
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.never;
+
+@WebMvcTest(PlanetController.class)
+class PlanetControllerTest extends AbstractWebMvcTest {
+
+    @MockBean
+    private PlanetService service;
+
+    @Test
+    void testGet() throws Exception {
+        // arrange
+        UUID id = UUID.randomUUID();
+        Planet planet = Planet.builder().name("name").build();
+        PlanetDto planetDto = modelMapper.map(planet, PlanetDto.class);
+        given(service.get(id)).willReturn(planet);
+        // act
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .get("/planet/get/{id}", id)
+                .accept(MediaType.APPLICATION_JSON);
+        MockHttpServletResponse response = mockMvc
+                .perform(requestBuilder)
+                .andReturn()
+                .getResponse();
+        // assert
+        verifySuccessfulResponse(response, planetDto);
+        then(service).should().get(id);
+    }
+
+    //@Test
+    //void testGet_entityNotFound() throws Exception {
+    //    // arrange
+    //    UUID id = UUID.randomUUID();
+    //    given(service.get(id)).willThrow(new AppException(ErrorCodeType.ENTITY_NOT_FOUND));
+    //    // act
+    //    RequestBuilder requestBuilder = MockMvcRequestBuilders
+    //            .get("/planet/get/{id}", id)
+    //            .accept(MediaType.APPLICATION_JSON);
+    //    MockHttpServletResponse response = mockMvc
+    //            .perform(requestBuilder)
+    //            .andReturn()
+    //            .getResponse();
+    //    // assert
+    //    verifyErrorResponse(response.getContentAsString(), ErrorCodeType.ENTITY_NOT_FOUND);
+    //    then(service).should().get(id);
+    //}
+
+    @Test
+    void testGetList() throws Exception {
+        // arrange
+        List<Planet> planets = List.of(Planet.builder().name("name").build());
+        List<PlanetDto> planetDtos = modelMapper.map(planets, new TypeToken<List<PlanetDto>>() {
+        }.getType());
+        given(service.getList()).willReturn(planets);
+        // act
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .get("/planet/get-list")
+                .accept(MediaType.APPLICATION_JSON);
+        MockHttpServletResponse response = mockMvc
+                .perform(requestBuilder)
+                .andReturn()
+                .getResponse();
+        // assert
+        verifySuccessfulResponse(response, planetDtos);
+        then(service).should().getList();
+    }
+
+    @Test
+    void testAdd() throws Exception {
```

Comment:
This test doesn't check dirty input fixes.

Reply:
stopped here

Reply:
remove @BeforeAll